_data/es/navigation.yml

```diff
@@ -1,17 +1,15 @@
 nav:
 - title:                 Cómo usar Jamulus
   subfolderitems:
-    - page:              Cómo Empezar
-      url:               /wiki/Getting-Started
+    - page:              Configuración
+      url:               /wiki/Setup
```

Comment:
Not sure, but I think @gilgongo would like to change it back to Getting-Started. But I think he'll do it close to the release.

Reply:
Yes, I was going to just find/replace all the links to /wiki/Setup to /wiki/Getting-Started once I've moved the Setup content into that file so we can then remove *-Setup.md and have one less redirect.

Reply:
Ok. So this means that we can just merge this as is